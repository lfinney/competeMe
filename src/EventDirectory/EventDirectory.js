import React from 'react';
import Event from '../Event/Event';

const EventDirectory = () => {
  // const events = //map over array of values from firebase

  return (
    <div className="EventDirectory">
      {/* {events} */}
    </div>
  );
};


export default EventDirectory;
