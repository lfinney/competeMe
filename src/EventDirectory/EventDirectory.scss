@import '../variables.scss';

.EventDirectory {
  width: 90%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin: auto;
}
