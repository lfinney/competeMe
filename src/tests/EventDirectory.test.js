import React from 'react';
import ReactDOM from 'react-dom';
import EventDirectory from '../EventDirectory/EventDirectory';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<EventDirectory />, div);
});
