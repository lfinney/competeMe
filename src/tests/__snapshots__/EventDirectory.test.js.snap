// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventDirectory should render component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventDirectory
    competitions={
        Array [
            Object {
              "compName": "Frisbee",
              "competitiveness": "casual",
              "date": "1987-10-09",
              "details": "This will be fun",
              "location": "Washington Park",
              "player": 7,
              "sport": "Frisbee",
              "time": "15:00",
            },
            Object {
              "compName": "Frisbee",
              "competitiveness": "casual",
              "date": "1987-10-09",
              "details": "This will be fun",
              "location": "Washington Park",
              "player": 7,
              "sport": "Frisbee",
              "time": "15:00",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Event
          activePopup={undefined}
          activeUser={undefined}
          comp={
                    Object {
                              "compName": "Frisbee",
                              "competitiveness": "casual",
                              "date": "1987-10-09",
                              "details": "This will be fun",
                              "location": "Washington Park",
                              "player": 7,
                              "sport": "Frisbee",
                              "time": "15:00",
                            }
          }
          liveUser={undefined}
          userCompetitions={undefined}
/>,
        <Event
          activePopup={undefined}
          activeUser={undefined}
          comp={
                    Object {
                              "compName": "Frisbee",
                              "competitiveness": "casual",
                              "date": "1987-10-09",
                              "details": "This will be fun",
                              "location": "Washington Park",
                              "player": 7,
                              "sport": "Frisbee",
                              "time": "15:00",
                            }
          }
          liveUser={undefined}
          userCompetitions={undefined}
/>,
      ],
      "className": "EventDirectory",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "activePopup": undefined,
          "activeUser": undefined,
          "comp": Object {
            "compName": "Frisbee",
            "competitiveness": "casual",
            "date": "1987-10-09",
            "details": "This will be fun",
            "location": "Washington Park",
            "player": 7,
            "sport": "Frisbee",
            "time": "15:00",
          },
          "liveUser": undefined,
          "userCompetitions": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "activePopup": undefined,
          "activeUser": undefined,
          "comp": Object {
            "compName": "Frisbee",
            "competitiveness": "casual",
            "date": "1987-10-09",
            "details": "This will be fun",
            "location": "Washington Park",
            "player": 7,
            "sport": "Frisbee",
            "time": "15:00",
          },
          "liveUser": undefined,
          "userCompetitions": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Event
            activePopup={undefined}
            activeUser={undefined}
            comp={
                        Object {
                                    "compName": "Frisbee",
                                    "competitiveness": "casual",
                                    "date": "1987-10-09",
                                    "details": "This will be fun",
                                    "location": "Washington Park",
                                    "player": 7,
                                    "sport": "Frisbee",
                                    "time": "15:00",
                                  }
            }
            liveUser={undefined}
            userCompetitions={undefined}
/>,
          <Event
            activePopup={undefined}
            activeUser={undefined}
            comp={
                        Object {
                                    "compName": "Frisbee",
                                    "competitiveness": "casual",
                                    "date": "1987-10-09",
                                    "details": "This will be fun",
                                    "location": "Washington Park",
                                    "player": 7,
                                    "sport": "Frisbee",
                                    "time": "15:00",
                                  }
            }
            liveUser={undefined}
            userCompetitions={undefined}
/>,
        ],
        "className": "EventDirectory",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "activePopup": undefined,
            "activeUser": undefined,
            "comp": Object {
              "compName": "Frisbee",
              "competitiveness": "casual",
              "date": "1987-10-09",
              "details": "This will be fun",
              "location": "Washington Park",
              "player": 7,
              "sport": "Frisbee",
              "time": "15:00",
            },
            "liveUser": undefined,
            "userCompetitions": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "activePopup": undefined,
            "activeUser": undefined,
            "comp": Object {
              "compName": "Frisbee",
              "competitiveness": "casual",
              "date": "1987-10-09",
              "details": "This will be fun",
              "location": "Washington Park",
              "player": 7,
              "sport": "Frisbee",
              "time": "15:00",
            },
            "liveUser": undefined,
            "userCompetitions": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
